pkg/labels/selector.go

```diff
@@ -809,3 +809,73 @@ func SelectorFromSet(ls Set) Selector {
 func ParseToRequirements(selector string) ([]Requirement, error) {
 	return parse(selector)
 }
+
+// Conflicts takes 2 maps
+// returns true if there a key match between the maps but the value doesn't match
+// returns false in other cases
+func Conflicts(labels1, labels2 map[string]string) bool {
+	for k, v := range labels1 {
+		if val, match := labels2[k]; match {
+			if val != v {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+// Merge combines given maps
+// Note: It doesn't not check for any conflicts between the maps
```

Comment:
Here it is just to find out if a conflict occurs (true or false) that means if  a same key exists, then their values are same or not. Not sure what do you mean by win here?